lte/gateway/deploy/roles/magma/files/systemd/magma_envoy.service

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+[Unit]
+Description=Magma envoy service
+PartOf=magma@envoy_controller.service
+Before=magma@envoy_controller.service
```

Comment:
I think we need to start magma@envoy_controller.service before start envoyD to avoid connection retry from envoyD to the controller.